Crypto/ECC/Simple/Prim.hs

```diff
@@ -49,7 +50,7 @@ pointNegate :: Curve curve => Point curve -> Point curve
 pointNegate        PointO     = PointO
 pointNegate point@(Point x y) =
     case curveType point of
-        CurvePrime {}  -> Point x (-y)
+        CurvePrime (CurvePrimeParam p) -> Point x (p - y)
```

Comment:
same remark as the other pointNegate call. I think it's clearly more accurate this way and keep y in the `Fe` range (maybe normalized would be the right term)